app/javascript/modules/listings/DirectoryHelpers.tsx

```diff
@@ -194,7 +195,7 @@ export const getListingCards = (listings, directoryType, stackedDataFxn, hasFilt
                 tableHeader: { text: getTableHeader(listing) },
                 tableSubheader: { text: getTableSubHeader(listing) },
                 contentHeader: { text: listing.Name },
-                contentSubheader: { text: getListingAddressString(listing) },
+                contentSubheader: { element: <ListingAddress listing={listing} />, text: "" },
```

Comment:
Won't let me comment on line 21, but looks like we should cleanup this import
`/home/circleci/web/app/javascript/modules/listings/DirectoryHelpers.tsx
  21:38  error  'getListingAddressString' is defined but never used  unused-imports/no-unused-imports`